packages/celotool/src/lib/geth.ts

```diff
@@ -859,6 +859,10 @@ export async function startGeth(
   const privateKey = instance.privateKey || ''
   const lightserv = instance.lightserv || false
   const etherbase = instance.etherbase || ''
+  // Allow maxPeers to be zero
+  const maxPeers = instance.maxPeers !== undefined ? instance.maxPeers : gethConfig.instances.length
+  // Default to true
+  const setNodeKey = instance.setNodeKey !== undefined ? instance.setNodeKey : true
```

Comment:
You should be able to use the [null coalescing operator](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-7.html#nullish-coalescing) (`??`) here.